token/cli/src/output.rs

```diff
@@ -176,6 +187,9 @@ impl fmt::Display for CliMultisig {
 #[serde(rename_all = "camelCase")]
 pub(crate) struct CliTokenAccount {
     pub(crate) address: String,
+    pub(crate) program_id: String,
+    pub(crate) epoch: u64,
+    pub(crate) decimals: Option<u8>,
```

Comment:
Unless I'm misreading something, this should never be `None` during creation, so you should be able to `unwrap()` and have this just as a `u8`

Reply:
its also created in sort.rs#60, which has less information to work with but uses almost none of it. oh which reminds me, i wanted to improve `spl-token accounts` too...

Reply:
Ah right, my mistake!